lib/auth/Vault.ts

```diff
@@ -221,6 +222,7 @@ export default class Vault {
                 logger: params.log,
                 securityToken: params.data.securityToken,
                 requestContext: serializedRCs,
+                ...options,
```

Comment:
may be better to put these before reqUid, logger, ... so that these field cannot be overriden by mistake:

```suggestion
           {
                ...options,
                // @ts-ignore
                reqUid: params.log.getSerializedUids(),
                logger: params.log,
                securityToken: params.data.securityToken,
                requestContext: serializedRCs,
           }
```